src/main/java/guicommand/UserIcon.java

```diff
@@ -0,0 +1,47 @@
+package guicommand;
+
+import javafx.scene.image.Image;
+import javafx.stage.FileChooser;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+public class UserIcon {
+
+    private Image icon;
+
+    public UserIcon() throws IOException {
+        FileReader fileReader = new FileReader("data/iconPath.txt");
+        BufferedReader bufferedReader = new BufferedReader(fileReader);
+        String iconPath = bufferedReader.readLine();
+        icon = new Image(this.getClass().getResourceAsStream(iconPath));
+        bufferedReader.close();
+    }
+
+    public Image getIcon() {
+        return icon;
+    }
+
+    public void changeIcon() throws IOException {
+        FileWriter fileWriter = new FileWriter("data/iconPath.txt", false);
+        BufferedWriter bufferedWriter = new BufferedWriter(fileWriter);
+        FileChooser chooser = new FileChooser();
+        chooser.setTitle("Select a picture:");
+        File defaultDirectory = new File("D:/");
```

Comment:
Remember that your product needs to be cross-platform compatible, as specified in the project constraints.